src/null-safety/unsound-null-safety.md

```diff
@@ -3,12 +3,23 @@ title: Unsound null safety
 description: Mixing language versions lets you migrate to null safety at your own pace, with some of the benefits of null safety.
 ---
 
-A Dart program can contain some libraries that
+A Dart program may contain some libraries that
 are [null safe][] and some that aren't.
 These **mixed-version programs**
-execute with **unsound null safety**.
+rely on **unsound null safety**.
+
+{{site.alert.warn}}
+Dart 3--planned for a mid-2023 release--
+will require sound null safety. It will prevent code from running without
```

Comment:
Use three dashes for an [em dash](https://developers.google.com/style/dashes) here instead:

```suggestion
Dart 3---planned for a mid-2023 release---will require sound null safety. 
It will prevent code from running without
```